### PR TITLE
- adds camtasia 19

### PR DESCRIPTION
**Pull request by baywet**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate <manifest>`, where `<manifest>` is the name of the manifest you're submitting?
- [x] Have you tested your manifest locally with `winget install -m <manifest>`?

-----

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/2615)

### CODE REVIEW

**Comment by baywet** [comment]
It looks like the CDN endpoint I'm hitting and the one the agent is using are not in sync. I'm not sure which one is correct though. Having a similar issue with #2616 Help please?

File: manifests/TechSmith/Camtasia/2019.0.10.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+License: Copyright (C) 2020 TechSmith
+MinOSVersion: 10.0.0.0
+Homepage: https://www.techsmith.com/
+Description: Screen Capture and Recording Software
+Tags: "snagit,screen capture,recording"
+Installers: 
+  - Arch: x64
+    Url: https://download.techsmith.com/camtasiastudio/releases/2006/camtasia.exe
+    InstallerType: exe
+    Sha256: 5049d55479143da18af1fcb6d2bdd88fd051fa086a4f16f686a623e4093db6a0
```
Comment (baywet) [inline manifests/TechSmith/Camtasia/2019.0.10.yaml:15]: This is the Sha I get for version 2020.0.6. Which is not the version of this YAML file (2019.0.10)

Comment (baywet) [reply manifests/TechSmith/Camtasia/2019.0.10.yaml:15]: CC @KevinLaMS

**Review by ghost — DISMISSED** [review]
Validation has completed

**Comment by KevinLaMS** [comment]
this is now working, but as it is a draft it cannot be merged.  maybe this is a duplicate now?

**Comment by baywet** [comment]
This is the Sha I get for version 2020.0.6. Which is not the version of this YAML file (2019.0.10)

**Comment by KevinLaMS** [comment]
if you are looking for the 2019.0.10 version, then you need a different URL.
this appears to be the URL you need.
https://download.techsmith.com/camtasiastudio/releases/19010/camtasia.exe

**Comment by baywet** [comment]
I know that, you're the one who set the URL and the hash to 2020.0.6 https://github.com/microsoft/winget-pkgs/commit/04a4dc3a474e69fa4928e4cd530a78ca896b2f25

**Comment by KevinLaMS** [comment]
Yikes.  Sorry.

**Review by ghost — APPROVED** [review]
Validation has completed

**Pull request merged**